src/components/m-button/mixins/_button.scss

```diff
@@ -201,7 +201,9 @@
 
     display: block;
 
-    width: 0;
+    // fix iOS double tap for :hover
+    // zero width is considered hidden
+    width: 0.001px;
```

Comment:
@LucaMele 
@TheDadi 
It would be possible to target these styles only at ios devices.
http://stephen.io/mediaqueries/

Reply:
And computation of decimal pixel values should be tested:
http://cruft.io/posts/percentage-calculations-in-ie/

Reply:
@LucaMele 
@TheDadi 
@sventschui 
According to my manual browserstack tests, decimal point rounding does not seem to be an issue :)

Reply:
I will run few tests and then ok :)